### PR TITLE
Modified README to contain information on the push-docker-image command

### PR DESCRIPTION
**Pull request by jacksondelametter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,7 +97,22 @@ mvn orphedomos:multiplatform-build-deploy -Dorphedomos.build.architecture=linux/
   <targetArchitectures>linux/arm64</targetArchitectures>
   <targetArchitectures>linux/amd64</targetArchitectures>
 </targetArchitectures>
+
+
+```
+
+#### Push Docker Image
+
+This command leverages the Docker daemon to push the generated Docker image to a repository. In order to use this command. The generated Docker image must already be loaded into Docker. The command cannot be used directly on a Docker image tarball.
```
Comment (d-ryan-ashcraft) [inline README.md]: S (Substantive): this is a little hard to follow.  Also I think you want a comma after command?

File: README.md [hunk 2]
```diff
@@ -96,7 +97,22 @@ mvn orphedomos:multiplatform-build-deploy -Dorphedomos.build.architecture=linux/
   <targetArchitectures>linux/arm64</targetArchitectures>
   <targetArchitectures>linux/amd64</targetArchitectures>
 </targetArchitectures>
+
+
+```
```
Comment (d-ryan-ashcraft) [inline README.md:99]: A (Administrative): here and in subsequent locations, I'd remove the new lines in these blocks. Let markdown format it and add spacing so it is consistent.

**Review by d-ryan-ashcraft — APPROVED** [review]
Thanks for improving the documentation!  A couple very minor notes inline.

File: README.md (hunk 2 above)
Comment (jacksondelametter) [reply README.md:99]: Fixed.

File: README.md (hunk 1 above)
Comment (jacksondelametter) [reply README.md]: Changed to have a simpler description.
